src/patternfly/components/Title/styles.scss

```diff
@@ -0,0 +1,119 @@
+$pf-c-title--xl--FontWeight: $pf-global--FontWeight--normal !default;
+
+// lg
+$pf-c-title--lg--MarginTop: $pf-global--spacer--lg !default;
+$pf-c-title--lg--MarginBottom: $pf-global--spacer--md !default;
+$pf-c-title--lg--LineHeight: $pf-global--LineHeight--md !default;
+$pf-c-title--lg--FontSize: $pf-global--FontSize--lg !default;
+$pf-c-title--lg--FontWeight: $pf-global--FontWeight--semi-bold !default;
+
+// md
+$pf-c-title--md--MarginTop: $pf-global--spacer--lg !default;
+$pf-c-title--md--MarginBottom: $pf-global--spacer--md !default;
+$pf-c-title--md--LineHeight: $pf-global--LineHeight--md !default;
+$pf-c-title--md--FontSize: $pf-global--FontSize--md !default;
+$pf-c-title--md--FontWeight: $pf-global--FontWeight--semi-bold !default;
+
+// sm
+$pf-c-title--sm--MarginTop: $pf-global--spacer--lg !default;
+$pf-c-title--sm--MarginBottom: $pf-global--spacer--xs !default;
+$pf-c-title--sm--LineHeight: $pf-global--LineHeight--md !default;
+$pf-c-title--sm--FontSize: $pf-global--FontSize--md !default;
+$pf-c-title--sm--FontWeight: $pf-global--FontWeight--semi-bold !default;
+
+.pf-c-title {
+  &.pf-is {
+    &-xxxxl {
+      font-size: $pf-c-title--xxxxl--FontSize;
+      font-weight: $pf-c-title--xxxxl--FontWeight;
+      line-height: $pf-c-title--xxxxl--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--xxxxl--MarginTop;
+        margin-bottom: $pf-c-title--xxxxl--MarginBottom;
+      }
+    }
+    &-xxxl {
+      font-size: $pf-c-title--xxxl--FontSize;
+      font-weight: $pf-c-title--xxxl--FontWeight;
+      line-height: $pf-c-title--xxxl--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--xxxl--MarginTop;
+        margin-bottom: $pf-c-title--xxxl--MarginBottom;
+      }
+    }
+    &-xxl {
+      font-size: $pf-c-title--xxl--FontSize;
+      font-weight: $pf-c-title--xxl--FontWeight;
+      line-height: $pf-c-title--xxl--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--xxl--MarginTop;
+        margin-bottom: $pf-c-title--xxl--MarginBottom;
+      }
+    }
+    &-xl {
+      font-size: $pf-c-title--xl--FontSize;
+      font-weight: $pf-c-title--xl--FontWeight;
+      line-height: $pf-c-title--xl--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--xl--MarginTop;
+        margin-bottom: $pf-c-title--xl--MarginBottom;
+      }
+    }
+    &-lg {
+      font-size: $pf-c-title--lg--FontSize;
+      font-weight: $pf-c-title--lg--FontWeight;
+      line-height: $pf-c-title--lg--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--lg--MarginTop;
+        margin-bottom: $pf-c-title--lg--MarginBottom;
+      }
+    }
+    &-md {
+
+      font-size: $pf-c-title--md--FontSize;
+      font-weight: $pf-c-title--md--FontWeight;
+      line-height: $pf-c-title--md--LineHeight;
+      &.pf-has-margin {
+        margin-top: $pf-c-title--md--MarginTop;
+        margin-bottom: $pf-c-title--md--MarginBottom;
+      }
```

Comment:
To go along with my previous comment, I wonder if these margin values should be pulled out to just be for `.pf-c-title.pf-has-margin`? If in most cases, I'm going to get the font size, font weight, and line height defined here by default without using pf-is-md, then it would be a pain to add that, just to get the margins.

Reply:
so we would remove md and replace with default. like this:
```<h1 class="pf-c-title pf-has-margin">default title</h1>```